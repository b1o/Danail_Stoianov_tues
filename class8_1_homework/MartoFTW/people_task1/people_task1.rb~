require "csv"

out_arr = []

	CSV.foreach(ARGV[0]) do |row|

		p pattern = row[2].split(" ")[0]

		if pattern.to_i <= ARGV[1].to_i
			out_arr << row 
		end
	end

	CSV.open("people_task1_result.csv","w") do |csv|
		out_arr.sort_by{|a,b,c,d| a}.each do |elem|
			csv << elem
		end
	end
	
