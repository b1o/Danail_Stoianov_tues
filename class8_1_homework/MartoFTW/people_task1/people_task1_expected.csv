Bruce Lee,1.80,70 kg,30
Chavdar Spasov,1.60,45 kg,50
Kosta Trifonov,1.69,67 kg,30
Malcho Drebchev,1.50,49 kg,23
Pavel Todorov,1.70,75 kg,15
Teodor Slavov,1.73,74 kg,25
